qiskit_experiments/framework/composite/composite_experiment.py

```diff
@@ -54,6 +54,12 @@ def __init__(
     def circuits(self):
         pass
 
+    def set_transpile_options(self, **fields):
+        super().set_transpile_options(**fields)
```

Comment:
Do you think composite experiment is stateful for transpile options? Or is it just for retrieving?

Reply:
I think it should be stateful